vagrant/base.sh

```diff
@@ -20,38 +20,41 @@ set -ex
 # If you update this, also update tests/docker/Dockerfile
 export KIBOSH_VERSION=d85ac3ec44be0700efe605c16289fd901cfdaa13
 
-if [ -z `which javac` ]; then
-    apt-get -y update
-    apt-get install -y software-properties-common python-software-properties
-    add-apt-repository -y ppa:webupd8team/java
-    apt-get -y update
+path_to_jdk_cache() {
+  jdk_version=$1
+  echo "/var/cache/oracle-jdk8-installer/jdk-${jdk_version}-linux-64.tar.gz"
```

Comment:
oh, and i forgot to mention what i *really* noticed originally, which is that this path has a fixed `oracle-jdk8-installer` in it, which doesn't match the `jdk_version` variable.

also, the path was originally meant to match https://github.com/apache/kafka/blob/5040ec02ddb3f14aa02d66402c2408a21b77a257/Vagrantfile#L85, which if we're moving away from due to moving away from the webupd8team installler, we can also clean up.

we could use a fixed `/tmp` directory here if we wanted to now since not using the installer means it doesn't need a fixed path and the jdk download filenames should all be unique

Reply:
I just moved the download to /tmp